docs/en/user-guide/editing/editing.rst

```diff
@@ -5,23 +5,23 @@ Editing of existing data
 
 This represents a guide on how to edit existing data in TWW.
 
-Demo project
+Demo project (Chapter not adjusted to TWW)
 ------------
 
 * Make sure you have imported the demo project with pgAdminIII (see the :ref:`database-initialization` chapter)
 * Open the demo project by going to the main menu and press **Project** --> **Open**  or by pressing ``CTRL``
   + ``O`` at the same time.
 
-.. figure:: images/tww_demoproject_selection_tww_en_qgs.jpg
+.. figure:: images/qgep_demoproject_selection_qgep_en_qgs.jpg
    :scale: 50 %
 
 * Login to the database
 
-.. figure:: images/tww_demoproject_login.jpg
+.. figure:: images/qgep_demoproject_login.jpg
 
 * You should now have the demo project loaded
 
-.. figure:: images/tww_demoproject_overview.jpg
+.. figure:: images/qgep_demoproject_overview.jpg
```

Comment:
Rename the image instead of the path

Reply:
OK, I changed the filenames. Imo it's the wrong place to write about the demoproject, but let it be now.
The pics should be changed if we have the new demoproject.